src/components/molecules/navSearch/NavSearch.tsx

```diff
@@ -161,25 +161,43 @@ export const NavSearch = () => {
       </div>
 
       {/* Results */}
-      {showResults && (
-        <div className="absolute top-0 left-0 right-0 border border-border-lighter rounded-xl bg-surface-light shadow-[0px_4px_48px_0px_rgba(0,0,0,0.08)] p-4 pt-16">
-          {/* Geographies */}
-          {geographyResults.length > 0 && (
-            <div className="my-6 flex flex-col gap-4">
-              <h3 className="text-text-brand text-sm font-medium select-none">Geographies</h3>
-              {geographyResults.map((geography) => (
-                <LinkWithQuery href={`/geographies/${geography.slug}`} key={geography.id} className="text-sm hover:underline">
-                  {withBoldMatch(geography.display_value, searchText)}
-                </LinkWithQuery>
-              ))}
+      {isFocused && (
+        <div className="absolute top-0 left-0 right-0 min-h-[56px] outline -outline-offset-1 outline-border-lighter rounded-xl bg-surface-light shadow-[0px_4px_48px_0px_rgba(0,0,0,0.08)]">
+          {showResults && (
+            <div className="flex flex-col gap-3 p-2 pt-[56px]">
+              {/* Geographies */}
+              {geographyResults.length > 0 && (
+                <div>
+                  <h3 className="px-2.5 py-1.5 mb-1 text-text-brand text-sm font-medium select-none">Geographies</h3>
+                  {geographyResults.map((geography) => (
+                    <NavSearchSuggestion
+                      key={geography.id}
+                      href={`/geographies/${geography.slug}`}
+                      Icon={
+                        <LuArrowRight height="16" width="16" className="opacity-0 group-hover:opacity-100 text-text-brand transition duration-200" />
+                      }
+                      onClick={handleSuggestionClick}
+                    >
+                      {withBoldMatch(geography.display_value, searchText)}
+                    </NavSearchSuggestion>
+                  ))}
+                </div>
+              )}
+              {/* Search */}
+              <NavSearchSuggestion
+                href={searchHref}
+                Icon={<LuSearch height="16" width="16" />}
+                hint={
+                  <div className="text-xs text-text-tertiary font-[440]">
```

Comment:
Suggestion / appreciation: would be nice to standardify this kind of keyboard instruction / help text one day. Like it.